recipes-core/images/emlinux-image-common.inc

```diff
@@ -0,0 +1,7 @@
+#
+# Copyright (c) Cybertrust Japan Co., Ltd.
+#
+# SPDX-License-Identifier: MIT
+#
+
+IMAGE_INSTALL:append = "${@bb.utils.contains('IMAGE_PREINSTALL', 'openssh-server', ' sshd-regen-keys', '', d)}"
```

Comment:
If an user customize the openssh-server package on their recipe,  user uses IMAGE_INSTALL variable instead of IMAGE_INSTALL. So we need  check IMAGE_INSTALL?

Reply:
> "${@bb.utils.contains('IMAGE_PREINSTALL', 'openssh-server', ' sshd-regen-keys', '', d)}"

We use `IMAGE_PREINSTALL` to install debian packages such as "openssh-server". To detect whether it is included needs to search the variable `IMAGE_PREINSTALL`.

> IMAGE_INSTALL:append =

On the other hands, `sshd-regen-keys` is a ISAR recipe. To add it needs to use `IMAGE_INSTALL`.

> If an user customize the openssh-server package on their recipe, ...

Is the point of your comment that this PR does not apply if the user write `openssh-server.bb` recipe?
I am not sure if it is possible to create a recipe that is the same as the Debian package name.